src/Sarif.Sarifer/BackgroundAnalysisService.cs

```diff
@@ -20,15 +21,24 @@ internal class BackgroundAnalysisService : IBackgroundAnalysisService
     {
 #pragma warning disable IDE0044, CS0649 // Provided by MEF
         [ImportMany]
-        private IEnumerable<IBackgroundAnalyzer> analyzers;
+        private readonly IEnumerable<IBackgroundAnalyzer> analyzers;
```

Comment:
>readonly [](start = 16, length = 8)

If you do this then you don't need one of the two pragmas (IDE0044 or CS0649). One of them, I don't remember which, is about the lack of `readonly`.